README.md

```diff
@@ -30,8 +30,11 @@ func main() {
 	// Add your own RPC URL here or use a public one
 	rpcUrl := "https://rpc-mumbai.maticvigil.com"
 
+	// Instantiate a new provider to pass into the SDK
+	provider, _ := ethclient.Dial(chainRpcUrl)
+
 	// Now you can create a new instance of the SDK to use
-	sdk, _ := thirdweb.NewThirdwebSDK(rpcUrl)
+	sdk, _ := thirdweb.NewThirdwebSDK(provider, "", "")
```

Comment:
I still think we need an options object with defaults here, this doesn't look clean at all